package.json

```diff
@@ -30,12 +30,14 @@
     "@skyux/indicators": "^3.0.0-rc.0",
     "@skyux/list-builder": "^3.0.0-rc.0",
     "@skyux/list-builder-view-checklist": "^3.0.0-rc.1",
+    "@skyux/list-builder-common": "^3.0.0-rc.1",
     "@skyux/modals": "^3.0.0-rc.0"
   },
   "dependencies": {},
   "devDependencies": {
     "@blackbaud/skyux": "2.30.0",
     "@blackbaud/skyux-builder": "1.28.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.5"
+    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.5",
+    "@skyux/list-builder-common": "^3.0.0-rc.1"
```

Comment:
We'll need to release skyux2 with common, first, then remove this from the dev dependencies.